widget/index.html

```diff
@@ -24,85 +24,89 @@
+                        <span class="margin--0">{{s.data.upVoteCount}} Vote</span>
+                        <span class="margin--0" ng-show="s.data.upVoteCount>1">s</span>
+                    </p>
                 </div>
-                <p class="pill text--secondary shrink--0" ng-click="expandVotes(s)">
-                    <span class="margin--0">{{s.data.upVoteCount}} Vote</span>
-                    <span class="margin--0" ng-show="s.data.upVoteCount>1">s</span>
+                <div class="suggestion padded padded--m padded--x">
+                    <h4 class="word--break" ng-show="s.data.title">{{s.data.title}}</h4>
+                    <p class="word--break margin-bottom--m">{{s.data.suggestion}}</p>
+                </div>
+                <div class="button__panel shrink--0 padded padded--sm padded--x">
+                    <button class="voteButton btn btn--icon--text btn--transparent" ng-click="upVote(s)"
+                        ng-class="{btnShadow:!s.disableUpvote,btntransparent:s.disableUpvote}">
+                        <span class="icon icon-thumbs-up2"></span>
+                        <span class="text-primary margin--0">Vote</span>
+                    </button>
+                    <button class="voteButton btn btn--icon--text btn--transparent" ng-click="goSocial(s)"
+                        ng-show="hasSocial">
+                        <span class="icon icon-bubble"></span>
+                        <span class="text-primary margin--0">Comment</span>
+                    </button>
+                </div>
+                <p class="name-list text--secondary ellipsis">
+                    <span class="text--secondary ellipsis" ng-repeat="u in s.voters">{{u.displayName}}</span>
                 </p>
             </div>
-            <div class="suggestion padded padded--m padded--x">
-                <h4 class="word--break" ng-show="s.data.title">{{s.data.title}}</h4>
-                <p class="word--break margin-bottom--m">{{s.data.suggestion}}</p>
-            </div>
-            <div class="button__panel shrink--0 padded padded--sm padded--x">
-                <button class="voteButton btn btn--icon--text btn--transparent" ng-click="upVote(s)"
-                    ng-class="{btnShadow:!s.disableUpvote,btntransparent:s.disableUpvote}">
-                    <span class="icon icon-thumbs-up2"></span>
-                    <span class="text-primary margin--0">Vote</span>
-                </button>
-                <button class="voteButton btn btn--icon--text btn--transparent" ng-click="goSocial(s)"
-                    ng-show="hasSocial">
-                    <span class="icon icon-bubble"></span>
-                    <span class="text-primary margin--0">Comment</span>
-                </button>
-            </div>
-            <p class="name-list text--secondary ellipsis">
-                <span class="text--secondary" ng-repeat="u in s.voters">{{u.displayName}}</span>
-            </p>
```

Comment:
Add container to fix add-button positioning
Template changes for persisting form data and dynamic user images